### PR TITLE
Questions Pankaj_Borade

### PR DESCRIPTION
**Pull request by BkPankaj**

### CODE REVIEW

**Comment by harshag37** [comment]
for 2nd question 
look for the given input data. Its something different than what you took
It will be accordingly
Example:
UP 20
the whole UP and 20 will be input

**Comment by harshag37** [comment]
For 3rd question
you can optimise your code. somewhere you are unnecessary increasing the time complexity.

File: QUESTION_1_Solution.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+
+if s.__len__() <= 12 and s.__len__() >= 6 :
+    print(s)
+    for x in s:
+        if  x == '0' or x == '1' or x == '2' or x == '3' or x == '4' or x == '5' or x == '6' or x == '7' or x == '8' or x == '9':
+            numeric = True
+
+
+        for i in range(65,90):
```
Comment (harshag37) [inline QUESTION_1_Solution.py]: here you can use a different mehod

**Comment by BkPankaj** [comment]
> for 2nd question look for the given input data. Its something different than what you took It will be accordingly Example: UP 20 the whole UP and 20 will be input

Sir, please check new pull request I have applied your condition

**Pull request merged**